python/dgl/graph.py

```diff
@@ -1181,7 +1194,7 @@ def _batcher(lst):
         if node_attrs is not None:
             # mapping from feature name to a list of tensors to be concatenated
             attr_dict = defaultdict(list)
-            for nid in range(self.number_of_nodes()):
+            for nid in nx_graph.nodes():
```

Comment:
Then revert this since the graph has been relabeled.

Reply:
Done.